.github/workflows/build.yaml

```diff
@@ -5,10 +5,21 @@ on:
 name: Build Chronos
 jobs:
   build:
+    strategy:
+      matrix:
+        version: [current, stable, beta]
     runs-on: [ windows-2019 ]
     steps:
       - uses: actions/checkout@v3
       - uses: microsoft/setup-msbuild@v1
+      - if: matrix.version == 'stable'
+        name: Use stable version of CEF
+        run: |
+          bash .github/rewrite-cef-version.sh ${{ matrix.version }}
```

Comment:
Maybe this comment is out of scope of this pull request, and just a question.

Do we need to rewrite `setup-cef.bat` ?
It seems that what we need to do is specifying `CEFVER` in `setup-cef.bat`.

So could we modify this as following?

* Set `CEFVER` from environment variables if specified in `setup-cef.bat`
  * Default: `cef_binary_111.2.1+g870da30+chromium-111.0.5563.64_windows32_minimal`
* Set environment variable `build.yaml` and don't rewrite  `setup-cef.bat`.

Reply:
@HashidaTKS 

I don't know where `setup-cef.bat` comes from, so hesitate to modify it at that time.
It seems that giving CEFVER via the environment variable is straightforward now, so could you change in another PR, please? after that PR is merged,  this PR also shold follow it.

Reply:
> so could you change in another PR, please? 

Sure, I'll try it now.

Reply:
I have created a new PR for specifying CEFVER via the environment variable in `setup-cef.bat`
#32